stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/list/SearchResultItem.kt

```diff
@@ -107,22 +108,33 @@ public fun SearchResultItem(
  * the message.
  *
  * @param searchResultItemState The state of the search result item.
+ * @param currentUser The currently logged in user.
  */
 @Composable
 internal fun DefaultSearchResultItemLeadingContent(
     searchResultItemState: ItemState.SearchResultItemState,
+    currentUser: User?,
 ) {
-    UserAvatar(
-        user = searchResultItemState.message.user,
-        modifier = Modifier
-            .padding(
-                start = ChatTheme.dimens.channelItemHorizontalPadding,
-                end = 4.dp,
-                top = ChatTheme.dimens.channelItemVerticalPadding,
-                bottom = ChatTheme.dimens.channelItemVerticalPadding,
+    (
+        searchResultItemState
+            .channel
+            ?.takeIf { it.members.size == 2 }
```

Comment:
Quick question about this logic: If we have a named group with exactly 2 members (`me` and `other`), search results of messages from `me` would look like:

(avatar of `other`) (`me` in `Group name`)

I was wondering if this is intentional, because the shown avatar would be different from the user who sent the message? (this I guess makes sense for direct messages, but I wanted to check if it is intentional also for 1-to-1 named groups)
Ex: 
![search_in_group](https://github.com/user-attachments/assets/47981d0c-f6a4-42e2-8457-a06d6636c5da)

Reply:
I think so.
At the end, even if the channel has name, it is a 1:1 conversation

Reply:
Got it, thank you!